### PR TITLE
home page changes

### PR DESCRIPTION
**Pull request by saloni-080601**

### CODE REVIEW

File: src/pages/Home/index.js [hunk 1]
```diff
@@ -238,6 +239,9 @@ function Home() {
       defalutPage = rolesLandingPages[userRole.key];
     }
   });
+  useEffect(() => {
+    history.push(defalutPage);
+  }, [defalutPage]);
```
Comment (Poonam-Singh-Bagh) [inline src/pages/Home/index.js:244]: For what purpose you're doing it?

File: src/components/Header/index.js [hunk 2]
```diff
@@ -241,19 +257,43 @@ function Header() {
...
     if (window.scrollY > 0) {
-      setElevation(9);
+      setElevation(8);
     } else {
       setElevation(0);
     }
   });
 
+  //   window.addEventListener("scroll", changeBackground);
+  let bgcolor = "linear-gradient(90deg, #C1DFC4 0%, #DEECDD 100%);";
```
Comment (Poonam-Singh-Bagh) [inline src/components/Header/index.js]: Remove this line

File: src/components/Header/index.js [hunk 3]
```diff
@@ -241,19 +257,43 @@ function Header() {
...
     <ThemeProvider theme={theme}>
       <AppBar
-        sx={{ zIndex: 10000 }}
-        position="sticky"
-        color="background"
         elevation={elevation}
+        sx={
+          bgColor
+            ? {
+                background: position ? bgcolor : "background",
```
Comment (Poonam-Singh-Bagh) [inline src/components/Header/index.js]: Add the color value in bgcolor instead `"linear-gradient(90deg, #C1DFC4 0%, #DEECDD 100%)"`

File: src/pages/Home/index.js (hunk 1 above)
Comment (saloni-080601) [reply src/pages/Home/index.js:244]: for adding a default pathway.

**Pull request merged**